cmd/apid/glide.yaml

```diff
@@ -8,3 +8,5 @@ import:
   version: master
 - package: github.com/30x/apidVerifyAPIKey
   version: master
+- package: github.com/30x/apidAnalytics
```

Comment:
Have a word with @alexkhimich on the glide.lock. I think the expectation is, you update the glide.lock to the right version, and the user will only do a glide install.